responses/nl/HassTurnOn.yaml

```diff
@@ -3,8 +3,8 @@ responses:
   intents:
     HassTurnOn:
       default: "{{ slots.name }} aangezet"
-      lights_area: De lampen in {{ slots.area }} zijn aangezet
-      fans_area: De ventilatoren in {{ slots.area }} zijn aangezet
+      lights_area: "De {{ slots.area | trim }}verlichting is aangezet"
```

Comment:
why do we need to trim? That is implying that "area" contains a space?

Reply:
Without `|  trim` I always had a space between area name and verlichting